README.md

```diff
@@ -1,30 +1,44 @@
-# GA_project3
+# Itinerary Generator App for GA_project3
 - This is a shared repository between Sandy and Songrong for GA_project3
 
 
-_Define the Purpose and Goals: Start by clearly defining the purpose of your app and the goals you want to achieve with it. Understand your target audience, their needs, and how your app can solve their problems._
+##Goal and purpose
+To design an AI-powered web-app that helps tourists plan their travel based on preferences such as travel days, food, and tour sight preferences, with the outcome displayed on Google Maps. 
 
-To design an app that helps tourists plan their travel based on preferences such as travel days, food, and tour sight preferences,with the outcome displayed in the Google map. 
+The goal is to simplify the travel planning process, save time, and enhance the overall travel experience for users.
 
 
 Research and Ideation: Conduct thorough research to gather insights about your target audience, competitors, and current market trends. Generate ideas and brainstorm potential features and functionalities for your app.
 
 TimeLine
 Stage 1 (Week 1): 
-Sandy to fill:
-Front-end Features
-
-- Create User account creation and user login journey (Front End: Sandy, BackEnd: SR)
-   - Sandy (todolist)
-   - Songrong (todolist)
-
-Songrong to fill: 
-- Backend database setup
- 
+Front-end
+- Wireframe of the web-app
+- Homepage and header with the following menu items:
+  Home/login 
+  Home/Saved trips [for users who are logged in]
+- User account creation and login via email
+- Trip creation
+- Allow users to have multiple saved trips with each containing its own details page containing the itinerary (stretch or week 3)
+- Email verification for sign-up (stretch goal?)
+- Login via Single Sign-Ons (Stretch goal)
+- Edit trip created in case of user input error (stretch goal)
+Backend
+- Database set-up
+
+Not sure where this falls -> streaming to make the response of openAI faster
 
 Stage 2 (Week 2):
+Front-end: 
+- Results generated by Chatgpt to display as 1 activity/per card
+- Option to save activity cards to trip/day
+- Option to remove and/or replace activity from trip/day 
+- Display activity cards on Google map
```

Comment:
Google Map has its own places card with details like opening hours, website etc. Will need to plan how the OpenAI results be stored. tbc